test/cpp/qps/driver.cc

```diff
@@ -623,6 +632,24 @@ std::unique_ptr<ScenarioResult> RunScenario(
   // the result.
   bool client_finish_first =
       (client_config.rpc_type() != STREAMING_FROM_SERVER);
+  
+  if (!prometheus_query_time_file.empty()) {
+    std::ofstream timestamps_outfile;
+    timestamps_outfile.open(prometheus_query_time_file);
+    
+    std::stringstream start_epoch_string;
+    start_epoch_string << start_epoch_time.count();
+    timestamps_outfile << "{\"start\": " + start_epoch_string.str() << ", ";
+
+    std::stringstream end_epoch_string;
```

Comment:
Nit: The name `end_epoch_string` is misleading since it's not a `std::string` instance, it's a `std::stringstream` instance. Maybe rename `end_epoch_ss` or just `end_epoch`?